### PR TITLE
ShA/add summary block to votes

### PR DESCRIPTION
**Pull request by AlekseiiShevchuk**

### CODE REVIEW

File: resources/assets/sass/styles.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 body {
   background: #f7f7f7;
 }
+.float-right {
```
Comment (binaryvo) [inline resources/assets/sass/styles.scss]: probably this style is not necessary. You can use bootstrap pull-right instead

File: resources/assets/templates/vote/voteHeader.hbs [hunk 2]
```diff
@@ -34,6 +34,27 @@
                 </ul>
             </div>
         {{/if}}
+
+        {{#if meta.isFinished}}
+        {{else}}
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteHeader.hbs]: {{#if meta.isFinished}}
{{else}}

please use {{#unless meta.isFinished}} instead

File: resources/assets/templates/vote/voteHeader.hbs [hunk 3]
```diff
@@ -34,6 +34,27 @@
...
+        <div>
+            {{#if model.summary}}
+            <div><hr><h3>{{model.summary}}</h3><hr></div>
+            {{else}}
+            <div><hr><h3>No summary yet</h3><hr></div>
```
Comment (binaryvo) [inline resources/assets/templates/vote/voteHeader.hbs]: No necessary to show 'No summary yet'. Just show nothing if no summary presented.

**Comment by binaryvo** [comment]
Please move summary below voting results

**Comment by binaryvo** [comment]
Please fix remarks and re-open PR

**Comment by AlekseiiShevchuk** [comment]
fixed

**Pull request merged**